### ISSUE TITLE
[a11y] Navigate to chapters through the Table of Contents

### ISSUE THREAD

**Issue by gregoriopellegrino** [readium/readium-desktop#156]
OS: Windows 10
Assistive technology: NVDA 2018.1.1

When on the TOC you click on a entry, the text of the chapter appears but the focus remains on the topbar: the hyperlink anchor should be focused

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/157

**Comment by danielweck** [comment]
TODO: fix in navigator (force keyboard focus via HTML / web API), so assigned to me.

**Comment by llemeurfr** [comment]
Also seen on Windows 10 build 1809 with Jaws 2019 Professional latest version (George)

**Comment by llemeurfr** [comment]
comment from G.Kercher: Real problem with using the TOC and getting to the chapter. I go to the table of contents and arrow down the long list of navigation doc items. I press enter, and nothing happens. I have to get back to the bookshelf view, press resume reading, go down to the cover link to that book, press enter, and that will finally get me to the right chapter. This needs to be simplified. If the person pressed on the TOC link and was taken directly there, this would be a huge improvement.
